Backend/Backend.WebServices/DataTransferObjects/EnterpriseDetailsDTO.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Backend.WebServices.DatabaseEntities;
+using System.Collections.Generic;
+using System.Data.Entity.Spatial;
+
+
+namespace Backend.WebServices.DataTransferObjects
+{
+    public class EnterpriseDetailsDTO
+    {
+        public EnterpriseDetailsDTO(DatabaseEntities.Enterprise enterprise)
+        {
+            Id = enterprise.Id;
+            Name = enterprise.Name;
+            //Password = enterprise.Password;
+            CoverImage = enterprise.CoverImageURL;
+            CoverCoordinate = enterprise.CoverCoordinate;
+            ModifiedUTC = enterprise.ModifiedUTC;
+            Participants = enterprise.Participants;
```

Comment:
I think the Participants list should be of type ParticipantDTO, and this line should be something like
`Participants = enterprise.Participants.Select(x => new ParticipantDTO(x)).ToList()`
And the other DTOs should do similar things for the child entities. Otherwise it will return an EnterpriseDTO with a list of Participant database objects :/